src-self-hosted/translate_c.zig

```diff
@@ -1394,6 +1405,21 @@ fn transDeclStmt(rp: RestorePoint, scope: *Scope, stmt: *const ZigClangDeclStmt)
                 node.semicolon_token = try appendToken(c, .Semicolon, ";");
                 try block_scope.block_node.statements.push(&node.base);
             },
+            .Typedef => {
+                const typedef_decl = @ptrCast(*const ZigClangTypedefNameDecl, it[0]);
+                const name = try c.str(ZigClangNamedDecl_getName_bytes_begin(
+                    @ptrCast(*const ZigClangNamedDecl, typedef_decl),
+                ));
+
+                const mangled_name = try block_scope.makeMangledName(c, name);
+                if (try checkForBuiltinTypedef(c, typedef_decl, mangled_name)) |ast_node| {
+                    return ast_node;
+                }
```

Comment:
This isn't correct and will cause an assertion failure when rendering this:
```c
int main(int argc, char **argv) {
  typedef int uint32_t;
}
```

To get same behavior as with global builtin typedefs, you should return a string from `checkForBuiltinTypedef` and add it to the list of aliases in `block_scope.variables`.

Reply:
How can you get a string that represents the aliased type? The code I recently pushed doesn't seem to have work as I intended.